modules/architecture-machine-roles.adoc

```diff
@@ -33,7 +33,7 @@ In a Kubernetes cluster, the master nodes run services that are required to cont
 Exactly three master nodes must be used for all production deployments.
 ====
 
-Services that fall under the Kubernetes category on the master include the Kubernetes API server, etcd, Kubernetes controller manager, and HAProxy services.
```

Comment:
@knobunc Can you confirm whether or not it's appropriate to remove "HAProxy services" from this sentence here?

We had HAProxy services as an entry in the table in 3.11 [1] but it's not in the table for 4.x [2].

[1] https://docs.openshift.com/container-platform/3.11/architecture/infrastructure_components/kubernetes_infrastructure.html#master
[2] https://docs.openshift.com/container-platform/4.7/architecture/control-plane.html#defining-masters_control-plane

Reply:
Per slack conversation with @knobunc, HAProxy services should be removed from this sentence.